modal/image.py

```diff
@@ -978,7 +978,7 @@ def from_registry(
         add_python: Optional[str] = None,
         **kwargs,
     ) -> "_Image":
-        """Build a Modal image from a public image registry, such as Docker Hub.
+        """Build a Modal image from a public or private image registry, such as Docker Hub.
```

Comment:
Maybe not directly related to your PR, but I'm confused about how you control the registry that this method points at.